docs/website/contents/for-developers/ReleaseProcess.md

```diff
@@ -27,11 +27,13 @@ EG If we were currently maintaining a 2.3 version and a 2.4 version simultaneous
 - Whenever we're maintaining only the one greatest-ever MAJOR.MINOR version of a package, then all work on that package will happen on the main branch.
 - Whenever we're also maintaining some lesser MAJOR.MINOR versions of a package, then some work for that package will also be happening on the respective _release branches_.
 
-A _release branch_ is a branch dedicated to the maintenance of some older release of a package.
+A _release branch_ is a branch dedicated to the maintenance of some older release of one or multiple packages.
 
-RULE: If we have released a package FOO-X.Y.Z, then the release-FOO-X.x.x branch MUST exist and its first-parent history MUST include the commit released as FOO-X.Y.Z.
-For example, when releasing version 2.0.0 of a package named FOO, we must create the branch named release-FOO-2.x.x.
-And when we later release 2.3.1 of FOO (and we've never released a 2.Y.Z with Y>3), then we'd need to advance the release-FOO-2.x.x branch to point at the commit being released as FOO-2.3.1.
+RULE: A release branch should branch off the main branch just after a released commit.
```

Comment:
Is it correct to say *after* or should it be *at*?

Reply:
I think *at* is better, thanks :+1: